packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -32,232 +37,252 @@ const FormAds = observer(() => {
+                                                        }
+                                                        onChange={e => {
+                                                            my_ads_store.restrictLength(e, handleChange);
+                                                        }}
+                                                        required
+                                                    />
+                                                )}
+                                            </Field>
+                                            <Field name='min_transaction'>
+                                                {({ field }) => (
+                                                    <Input
+                                                        {...field}
+                                                        data-lpignore='true'
+                                                        type='text'
+                                                        error={touched.min_transaction && errors.min_transaction}
+                                                        label={localize('Min order')}
+                                                        className='p2p-my-ads__form-field'
+                                                        trailing_icon={
+                                                            <span className='p2p-my-ads__form-field--trailing'>
+                                                                {currency}
+                                                            </span>
+                                                        }
+                                                        onChange={e => {
+                                                            my_ads_store.restrictLength(e, handleChange);
+                                                        }}
+                                                        required
+                                                    />
+                                                )}
+                                            </Field>
+                                            <Field name='max_transaction'>
+                                                {({ field }) => (
+                                                    <Input
+                                                        {...field}
+                                                        data-lpignore='true'
+                                                        type='text'
+                                                        error={touched.max_transaction && errors.max_transaction}
+                                                        label={localize('Max order')}
+                                                        className='p2p-my-ads__form-field'
+                                                        trailing_icon={
+                                                            <span className='p2p-my-ads__form-field--trailing'>
```

Comment:
it's better if we use the Text component for all 'p2p-my-ads__form-field--trailing' classes

Reply:
IIRC Sara was updating to Text component in her PRs?

Reply:
Text component is merged, we can use them now.

Reply:
Sara had a PR to move to Text component in P2P (https://github.com/binary-com/deriv-app/pull/2374) but I see that it's only for paragraph so I'll update the span here.